force-app/main/default/lwc/modal/modal.js

```diff
@@ -15,6 +15,21 @@ const CSS_CLASS = "modal-hidden";
 export default class Modal extends LightningElement {
     close = closeLabel;
 
+    @api size = "fixed";
+    visibilityClass = CSS_CLASS;
```

Comment:
Being nit-picky here but can we change CSS_CLASS to say css_modal_hidden_class since it is a constant that holds "modal-hidden"